packages/amplify-category-custom/package.json

```diff
@@ -26,6 +26,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@types/lodash": "^4.14.149",
```

Comment:
Can this be a devDependency?

Reply:
it's here
https://github.com/aws-amplify/amplify-cli/blob/e5f00c392b150bbb47aed30c81cb5982b17c51a2/packages/amplify-category-custom/src/utils/dependency-management-utils.ts#L6